src/text/line.rs

```diff
@@ -1,9 +1,11 @@
 #![allow(deprecated)]
 use super::{Span, Spans, Style};
+use crate::layout::Alignment;
 
 #[derive(Debug, Clone, PartialEq, Default, Eq)]
 pub struct Line<'a> {
-    pub spans: Vec<Span<'a>>,
+    spans: Vec<Span<'a>>,
+    pub alignment: Option<Alignment>,
 }
```

Comment:
Now that I see `pub alignment` here, I wonder I was wrong about whether `spans` should be private. Making it public seems to be more consistent to other structs (e.g. `Span`, `Text`, `Style` and many of the widget fields). What do you think?

Reply:
I am with you on this one. However, I believe that these structs should be **extendable** without every time implying a breaking change.

Such breaking change should not be a major deal though, since the amount of users directly initializing these structs should be minimal. So it might not be worth the effort and extra weirdness to circumvent this issue.

Since you are the crate maintainers, the decision ultimately is yours. If I were to decide, I would leave `spans` as is (public) and accept that this implies a minor breaking change.

Reply:
Ok, let's go with the public spans field here instead of the methods and then I think this is good to ship.